app/src/app/components/contents/downloads/Artefacts.tsx

```diff
@@ -0,0 +1,33 @@
+import { Card, CardContent, CardHeader } from "../../Base/Card";
+import FileRow from "./FileRow";
+import { Artefact, PacketMetadata } from "../../../../types";
+
+interface ArtefactsProps {
+  artefacts: Artefact[];
+  packet: PacketMetadata;
+}
+
+export default function Artefacts({ artefacts, packet }: ArtefactsProps) {
```

Comment:
If Artefacts and OtherFiles are orderly specific, maybe they should be named "OrderlyArtefacts" etc, or maybe we should have an orderly subfolder for orderly specific stuff. I've a feeling this might get a bit tangled between orderly and non-orderly implementations and would be good to keep them cleanly separated.